assembl/static2/css/views/search.scss

```diff
@@ -0,0 +1,346 @@
+@import "../variables.scss";
+@import "../../node_modules/searchkit/theming/vars.scss";
+// @import "../../node_modules/searchkit/theming/components.scss";
+$sk-active-bg-color: $purple;
+$sk-action-text-color: $purple;
+$sk-selector-active-background: $purple;
+$highlightColor: #d6c3f6;
+
+@import "../../node_modules/searchkit/theming/components/arrows.scss";
+@import "../../node_modules/searchkit/theming/components/actions.scss";
+@import "../../node_modules/searchkit/theming/components/loader.scss";
+
+// @import "../../node_modules/searchkit/theming/components/search-box.scss";
+@import "../../node_modules/searchkit/theming/components/search-icon.scss";
+
+@import "../../node_modules/searchkit/theming/components/hits-stats.scss";
+@import "../../node_modules/searchkit/theming/components/no-hits.scss";
+
+@import "../../node_modules/searchkit/theming/components/filter-group.scss";
+
+@import "../../node_modules/searchkit/theming/components/filter-item.scss";
+@import "../../node_modules/searchkit/theming/components/item-list.scss";
+@import "../../node_modules/searchkit/theming/components/select.scss";
+@import "../../node_modules/searchkit/theming/components/tabs.scss";
+@import "../../node_modules/searchkit/theming/components/tag-cloud.scss";
+@import "../../node_modules/searchkit/theming/components/toggle.scss";
+
+@import "../../node_modules/searchkit/theming/components/range-input.scss";
+@import "../../node_modules/searchkit/theming/components/range-histogram.scss";
+@import "../../node_modules/searchkit/theming/components/range-slider.scss";
+
+@import "../../node_modules/searchkit/theming/components/hierarchical-menu-filter.scss";
+@import "../../node_modules/searchkit/theming/components/hierarchical-refinement-filter.scss";
+@import "../../node_modules/searchkit/theming/components/input-filter.scss";
+@import "../../node_modules/searchkit/theming/components/numeric-refinement-list-filter.scss";
+@import "../../node_modules/searchkit/theming/components/refinement-list-filter.scss";
+@import "../../node_modules/searchkit/theming/components/selected-filters.scss";
+@import "../../node_modules/searchkit/theming/components/range-filter.scss";
+@import "../../node_modules/searchkit/theming/components/reset-filters.scss";
+@import "../../node_modules/searchkit/theming/components/menu-filter.scss";
+@import "../../node_modules/searchkit/theming/components/tag-filter.scss";
+
+@import "../../node_modules/searchkit/theming/components/pagination.scss";
+
+@import "../../node_modules/searchkit/theming/components/table.scss";
+
+@import "../../node_modules/searchkit/theming/components/panel.scss";
+
+
+.emoticonValue {
+    display: inline-block;
+    width: 20px;
+    height: 20px;
+    margin: 0 0 0 -14px;
+    padding:  2px 0 0 6px;
+    border-radius: 50%;
+    background-color: $highlightColor;
+}
+
+
+.sk-tag-filter {
```

Comment:
Please add a comment to here to say that this style customizes the rendering of the searchkit external module and that these CSS classes names are chosen by this library, because it's not self-explanatory when you are looking at it outside of the context of this commit/PR.